src/utils/datetime/formatters.ts

```diff
@@ -7,7 +7,7 @@ const dateTimeOptions: any = {
   day: '2-digit',
   month: '2-digit',
   year: 'numeric',
-  hour: 'numeric',
+  hour: '2-digit',
```

Comment:
Our validator is strict, so we need the hour to be always 2-digits. Example: `2:00` vs `02:00`

Reply:
This seems like potentially a design change. I wonder if anyone else has feedback on changing the hourly format to always to use 2 digits.

Reply:
this seems like it'll have an effect on formats like [this one](https://github.com/influxdata/ui/blob/master/src/visualization/utils/timeFormat.ts#L5) which use `am` and `pm`

`YYYY-MM-DD hh:mm:ss a ZZ`

which would be 1983-07-04 2:35:00 am EDT

will this change those formats?

Reply:
Yes, it will make them `1983-07-04 02:35:00 am EDT`